2/두정수사이의합.js

```diff
@@ -0,0 +1,11 @@
+function solution(a, b) {
+  let answer = 0;
+  let c = 0;
+  if (a > b) {
+    c = a;
+    a = b;
+    b = c;
```

Comment:
```suggestion
    c = a;
    a = b;
    b = c;
```
<참고>
Math.max()와 Math.min() 메서드를 활용하면 최대값과 최소값을 더 알아보기 쉽울 것 같아요.

Reply:
피드백 참고해서 코드 수정했습니다 !
for (let i = Math.min(a, b); i <= Math.max(a, b); i++) {
    answer += i;
}